### PR TITLE
feat: CapBox

### PR DESCRIPTION
**Pull request by jakubuid**
CapBox

### CODE REVIEW

File: protocol/notify/src/main/kotlin/com/walletconnect/notify/engine/calls/IsRegisteredUseCase.kt [hunk 1]
```diff
@@ -23,12 +23,18 @@ internal class IsRegisteredUseCase(
                     !allApps && registeredAccount.appDomain != domain -> false
                     allApps && !registeredAccount.allApps -> false
                     !allApps && registeredAccount.allApps -> false
-                    else -> identitiesInteractor.getAlreadyRegisteredValidIdentity(AccountId(account), Statement.fromBoolean(allApps).content, domain, listOf(identityServerUrl))
+                    else -> identitiesInteractor.getAlreadyRegisteredValidIdentity(
+                        accountId = AccountId(account),
+                        domain = domain,
+                        resources = listOf(identityServerUrl, createAuthorizationReCaps())
```
Comment (jakubuid) [inline protocol/notify/src/main/kotlin/com/walletconnect/notify/engine/calls/IsRegisteredUseCase.kt:29]: Q: Should the authorization ReCaps be added as well on "is already registered" flow?

Comment (chris13524) [reply protocol/notify/src/main/kotlin/com/walletconnect/notify/engine/calls/IsRegisteredUseCase.kt:29]: I think so?

Comment (jakubuid) [reply protocol/notify/src/main/kotlin/com/walletconnect/notify/engine/calls/IsRegisteredUseCase.kt:29]: That's what I think too

File: protocol/notify/src/main/kotlin/com/walletconnect/notify/engine/domain/Utils.kt [hunk 2]
```diff
@@ -0,0 +1,6 @@
+package com.walletconnect.notify.engine.domain
+
+fun createAuthorizationReCaps(): String {
+    // {"att":{"https://notify.walletconnect.com":{"manage/all-apps-notifications":[{}]}}}
```
Comment (TalhaAli00) [inline protocol/notify/src/main/kotlin/com/walletconnect/notify/engine/domain/Utils.kt:4]: Do we still need this line?

**Review by TalhaAli00 — APPROVED** [review]

**Pull request merged**